src/jit/emitfmtsxarch.h

```diff
@@ -194,6 +196,8 @@ IF_DEF(ARD_CNS,     IS_AM_RD,                   AMD_CNS)  // read  [adr], const
 IF_DEF(AWR_CNS,     IS_AM_WR,                   AMD_CNS)  // write [adr], const
 IF_DEF(ARW_CNS,     IS_AM_RW,                   AMD_CNS)  // r/w   [adr], const
 
+IF_DEF(AWR_RRD_CNS, IS_AM_WR,                   AMD_CNS)  // write  [adr], reg, const
```

Comment:
This hsould be `IS_AM_WR|IS_R1_RD`? and the comment should be `write [adr], read reg, const`?

Reply:
> This hsould be IS_AM_WR|IS_R1_RD?

I am not sure, this field seems not used now.

> and the comment should be write [adr], read reg, const?

Will change.

Reply:
I don't think a flag being unused is a good reason to drop it. We should probably add it, in order to match the others, and it c an be removed, ignored, or consumed (whoever is appropriate) by both current or future code/changes